src/components/Modal/ProposalPlugins.vue

```diff
@@ -97,6 +97,21 @@ export default {
       })
     );
   },
+  computed: {
+    modalProps() {
+      if (this.selected === 'safeSnap') {
+        return {
+          shellProps: {
+            style: 'maxWidth: 610px'
```

Comment:
I would prefer to keep a consistent width size for the modals. What do you think if both the configuration of the plugin and the preview is visible directly on the left side on proposal creation? Here: https://github.com/snapshot-labs/snapshot/pull/380/files#diff-e08c7980ee765157e2398025e81ad42e5426b886e0878dc919562b960a976794R80

Reply:
I moved the plugin configuration and transaction previews to the left side, this is how it currently looks:

![Screenshot from 2021-05-26 23-36-06](https://user-images.githubusercontent.com/2939980/119762239-51f1fb80-be7b-11eb-96d1-c8cb88ee7a82.png)